src/main/java/com/openshift/internal/restclient/capability/resources/OpenShiftBinaryPodLogRetrieval.java

```diff
@@ -39,10 +41,16 @@ public boolean isSupported() {
 	public String getName() {
 		return OpenShiftBinaryPodLogRetrieval.class.getSimpleName();
 	}
-
+	
 	@Override
 	public InputStream getLogs(boolean follow) {
+		return getLogs(follow, null);
+	}
+
+	@Override
+	public InputStream getLogs(boolean follow, String container) {
```

Comment:
Can this method be called concurrently, the second call changing the inner state while the first call has not yet requested build args?